lib/declare_schema/rake.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require "declare_schema"
+require "declare_schema/schema_change/all"
+require "declare_schema/schema_change/column_remove"
+require "generators/declare_schema/migration/migrator"
+require "erb"
+
+namespace :declare_schema do
+  desc 'Generate migrations for the database schema'
+  task :generate => 'db:load_config' do
+    up, down = Generators::DeclareSchema::Migration::Migrator.new(renames: {}).generate
+
+    if up.blank?
+      puts "Database and models match -- nothing to change"
+      return
+    end
+
+    puts "\n---------- Up Migration ----------"
+    puts up
+    puts "----------------------------------"
+
+    puts "\n---------- Down Migration --------"
+    puts down
+    puts "----------------------------------"
+
+    final_migration_name = default_migration_name
+    migration_template   = ERB.new(<<~EOF.chomp)
+      # frozen_string_literal: true
+      class <%= @migration_class_name %> < (ActiveRecord::Migration[4.2])
```

Comment:
BTW, I wonder if the `[4.2]` will cause us problems someday. I guess they plan to support it forever...but still, we might do better with a more modern format at some point.

Reply:
The parens here seem unnecessary:
```
< (ActiveRecord::Migration[4.2])
```
I'm guessing they were left over from when we had some ternary logic in there.

Reply:
Good point. I went ahead and switched this to use the same template that's used by the generators (duplicated this originally to implement the rake task outside of the gem). Also removed the unnecessary `()`s. Agree we should be mindful of the rails version in the migrations so that we can upgrade and start using more current functionality for rails migrations as well.

a86eb21682e2e7cb3bcedbb860c38288f52bf11b